src/components/Calculator.js

```diff
@@ -0,0 +1,69 @@
+import calculate from '../logic/calculate';
+
+function Calculator() {
+  const initialState = {
+    total: null,
+    next: null,
+    operation: null,
+  };
+
+  const [data, setData] = useState(initialState);
+
+  const getButtonName = (e) => {
+    const btnName = e.target.value;
+    const result = calculate(data, btnName);
+    setData(result);
+  };
+
+  return (
+    <div className="app_container">
+
+      <section className="result">
+        <p>
+          {data.total}
+          {data.operation}
+          {data.next}
+        </p>
+      </section>
+
+      <section className="quad">
+        <button value="A/C" type="button" onClick={getButtonName} className="normal_btn">A/C</button>
+        <button value="%" type="button" onClick={getButtonName} className="normal_btn negative_margin">+/-</button>
+        <button value="%" type="button" onClick={getButtonName} className="normal_btn negative_margin">%</button>
+        <button value="÷" type="button" onClick={getButtonName} className="orange_btn negative_margin">÷</button>
+      </section>
+
+      <section className="quad">
+        <button value="7" type="button" onClick={getButtonName} className="normal_btn">7</button>
+        <button value="8" type="button" onClick={getButtonName} className="normal_btn negative_margin">8</button>
+        <button value="9" type="button" onClick={getButtonName} className="normal_btn negative_margin">9</button>
+        <button value="*" type="button" onClick={getButtonName} className="orange_btn negative_margin">*</button>
+      </section>
+
+      <section className="quad">
+        <button value="4" type="button" onClick={getButtonName} className="normal_btn">4</button>
+        <button value="5" type="button" onClick={getButtonName} className="normal_btn negative_margin">5</button>
+        <button value="6" type="button" onClick={getButtonName} className="normal_btn negative_margin">6</button>
+        <button value="-" type="button" onClick={getButtonName} className="orange_btn negative_margin">-</button>
+      </section>
+
+      <section className="quad">
+        <button value="1" type="button" onClick={getButtonName} className="normal_btn">1</button>
+        <button value="2" type="button" onClick={getButtonName} className="normal_btn negative_margin">2</button>
+        <button value="3" type="button" onClick={getButtonName} className="normal_btn negative_margin">3</button>
+        <button value="+" type="button" onClick={getButtonName} className="orange_btn negative_margin">+</button>
+      </section>
+
+      <section className="quad">
+        <button value="0" type="button" onClick={getButtonName} className="normal_big_btn">0</button>
+        <button value="." type="button" onClick={getButtonName} className="normal_btn negative_margin">.</button>
+        <button value="=" type="button" onClick={getButtonName} className="orange_btn negative_margin">=</button>
+      </section>
+
+    </div>
```

Comment:
- I get this error when I try performing a multiply operation. Please try to fix this issue 🙏 
<img width="960" alt="ggwg" src="https://user-images.githubusercontent.com/73607512/219116597-4fbdbdd2-b393-42b1-ace1-69c2745af21b.PNG">